internal/controlplane/handlers_repositories_test.go

```diff
@@ -236,6 +236,7 @@ type EventPayload struct {
 	Repository int
 }
 
+// NOTE: this stub is also used by handlers_oauth_test.go ¯\_(ツ)_/¯
```

Comment:
seems we have a bunch of these all over the place. Besides this we have:
* https://github.com/stacklok/minder/blob/main/internal/eea/eea_test.go#L508-L535
* https://github.com/stacklok/minder/blob/main/internal/util/testqueue/passthroughqueue.go

Not as part of this PR, but it would be good to push these test utilities to a package that's easier to find.

Reply:
I'd love to move much of the implementations out of `controlplane/handlers_`, and I expect that normalizing these (or reducing their scope) will be part of that.